Framework/Geometry/src/Surfaces/Plane.cpp

```diff
@@ -306,14 +306,24 @@ int Plane::LineIntersectionWithPlane(V3D startpt, V3D endpt, V3D &output) {
   double sprod = this->getNormal().scalar_prod(startpt - endpt);
   if (sprod == 0)
     return 0;
-  double s1 = (NormV[0] * startpt[0] + NormV[1] * startpt[1] +
-               NormV[2] * startpt[2] - Dist) /
-              sprod;
+  double projection =
+      NormV[0] * startpt[0] + NormV[1] * startpt[1] + NormV[2] * startpt[2];
+  double s1 = (projection - Dist) / sprod;
   if (s1 < 0 || s1 > 1)
     return 0;
+  /*Action: deprecated expressions below. Reason: incur in numerical errors when
```

Comment:
Remove the old code and add a comment about why the new way is numerically correct

Reply:
done